src/useTitle.ts

```diff
@@ -5,21 +5,23 @@ export interface UseTitleOptions {
 }
 
 const DEFAULT_USE_TITLE_OPTIONS: UseTitleOptions = {
-  restoreOnUnmount: false,
+  restoreOnUnmount: true,
```

Comment:
it should be true as default, more we don't need this option, if user do not want restore, they could just do assign to `document.title` lol

wait for discussing with owner, and we can preserve(stay in false) or remove this option in later commit.